catmux/resources/example_session.yaml

```diff
@@ -1,18 +1,21 @@
 ---
+parameters:
+  # Parameters specified here can be used below and they can be overwritten at runtime
+  show_layouts: false # This parameter is used together with the if flag below
+  conditional_command: false
```

Comment:
I think, a more descriptive name would be good, so it is clear that the name is an arbitrary example, rather than a functionality. E.g. `print_additional_intro`.